### ISSUE TITLE
Replace PyQt imports with qtpy

### ISSUE THREAD

**Issue by jo-mueller** [jo-mueller/RadiAID#21]
importing PyQt in Python projects is one of the key ways to screw up repositories. It would be better to change all imports that rely on PyQt5 (which are A LOT in this project) to qtpy imports, which forms a compatibility layer above PyQt5 and should thus be more compatible with other platforms/computer/environments/etc

@schneidorlein, can I assign you to this? It would also be a nice exercise on how to work with github (I've only really learned it myself recently, tbh^^").

The conduct would be this:
- create a branch from the master branch
- do the changes necessary to fix this issue (and only these changes!)
- Send a pull request with the changes made (Github offers the "Compare and pull request" button of you go to your branch on the github page.
- We will then review the changes and merge it if all works :)

Let me know if you need any help with this!

**Comment by schneidorlein** [comment]
I can try this (#21 and #22), but I also do not have a lot of experience with Github yet :D
How urgent is it? ;)

**Comment by jo-mueller** [comment]
Ah - there's a comment :) Feel free to always add @jo-mueller to the comment, then I'll get notified.

**Comment by jo-mueller** [comment]
>How urgent is it? ;)

Not at all. Consider these tasks to be opportunities to get to know github a bit better (it's really powerful) within the scope of relatively small changes.

**Comment by jo-mueller** [comment]
Added qtpy to deps in #26
